test/specialty.jl

```diff
@@ -216,6 +216,15 @@ using ImageFiltering: IdentityUnitRange
         ImageFiltering.fillbuf_nan[] = false
         @test Kernel.LoG(2.5) == Kernel.LoG((2.5,2.5))
     end
+
+    @testset "moffat" begin
+        α = rand()
+        β = rand()
+        @test Kernel.moffat(α,β,2) == Kernel.moffat(α,β,(2,2))
+
+        fwhm = ceil(Int, (α*2*sqrt(2^(1/β) - 1)) * 4)
```

Comment:
There's nothing preventing `β` from being tiny here; this causes occasional test failures like [this one](https://travis-ci.org/JuliaImages/ImageFiltering.jl/jobs/658104201?utm_medium=notification&utm_source=github_status):
```
moffat: Error During Test at /Users/travis/build/JuliaImages/ImageFiltering.jl/test/specialty.jl:220

  Got exception outside of a @test

  InexactError: trunc(Int64, 1.4650326325253482e37)

  Stacktrace:

   [1] trunc at ./float.jl:696 [inlined]

   [2] ceil(::Type{Int64}, ::Float64) at ./float.jl:357

   [3] macro expansion at /Users/travis/build/JuliaImages/ImageFiltering.jl/test/specialty.jl:225 [inlined]

   [4] macro expansion at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.0/Test/src/Test.jl:1083 [inlined]

   [5] macro expansion at /Users/travis/build/JuliaImages/ImageFiltering.jl/test/specialty.jl:221 [inlined]

   [6] macro expansion at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.0/Test/src/Test.jl:1083 [inlined]

   [7] top-level scope at /Users/travis/build/JuliaImages/ImageFiltering.jl/test/specialty.jl:6

   [8] include at ./boot.jl:317 [inlined]

   [9] include_relative(::Module, ::String) at ./loading.jl:1044

   [10] include(::Module, ::String) at ./sysimg.jl:29

   [11] include(::String) at ./client.jl:392

   [12] top-level scope at none:0

   [13] include at ./boot.jl:317 [inlined]

   [14] include_relative(::Module, ::String) at ./loading.jl:1044

   [15] include(::Module, ::String) at ./sysimg.jl:29

   [16] include(::String) at ./client.jl:392

   [17] top-level scope at none:0

   [18] eval(::Module, ::Any) at ./boot.jl:319

   [19] exec_options(::Base.JLOptions) at ./client.jl:243

   [20] _start() at ./client.jl:425

Test Summary: | Pass  Error  Total

specialty     |   98      1     99

  Laplacian   |   63            63

  gaussian    |   25            25

  DoG         |    6             6

  LoG         |    3             3

  moffat      |    1      1      2
```
While the fix seems obvious, @gerrygralton I'd rather leave this to you since you know the `moffat` stuff much better than I do.

Reply:
And a different variant of the same theme:
```
moffat: Error During Test at /home/travis/build/JuliaImages/ImageFiltering.jl/test/specialty.jl:226

  Test threw exception

  Expression: Kernel.moffat(α, β) == Kernel.moffat(α, β, (fwhm, fwhm))

  OutOfMemoryError()

  Stacktrace:

   [1] Array at ./boot.jl:407 [inlined]

   [2] Array at ./boot.jl:415 [inlined]

   [3] Array at ./boot.jl:422 [inlined]

   [4] similar at /home/travis/.julia/packages/OffsetArrays/fZSaL/src/OffsetArrays.jl:124 [inlined]

   [5] similar at ./broadcast.jl:196 [inlined]

   [6] copy at ./broadcast.jl:840 [inlined]

   [7] materialize at ./broadcast.jl:820 [inlined]

   [8] moffat(::Float64, ::Float64, ::Tuple{Int64,Int64}) at /home/travis/build/JuliaImages/ImageFiltering.jl/src/kernel.jl:408

   [9] moffat at /home/travis/build/JuliaImages/ImageFiltering.jl/src/kernel.jl:410 [inlined]

   [10] moffat(::Float64, ::Float64) at /home/travis/build/JuliaImages/ImageFiltering.jl/src/kernel.jl:411

   [11] top-level scope at /home/travis/build/JuliaImages/ImageFiltering.jl/test/specialty.jl:226

   [12] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113

   [13] top-level scope at /home/travis/build/JuliaImages/ImageFiltering.jl/test/specialty.jl:221

   [14] top-level scope at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1113

   [15] top-level scope at /home/travis/build/JuliaImages/ImageFiltering.jl/test/specialty.jl:6

```